pkg/frontend/adminactions/azureactions.go

```diff
@@ -56,6 +61,11 @@ func NewAzureActions(log *logrus.Entry, env env.Interface, oc *api.OpenShiftClus
 		return nil, err
 	}
 
+	fpClientCertCred, err := env.FPNewClientCertificateCredential(subscriptionDoc.Subscription.Properties.TenantID)
```

Comment:
@cfsmikedrum @zacharyljones I think this is wrong. The AppLens' tenant ID is fixed per cloud:
https://msazure.visualstudio.com/CloudNativeCompute/_git/aks-rp?path=/resourceprovider/server/microsoft.com/containerservice/templates/implementation.go&version=GBmaster&line=645&lineEnd=645&lineStartColumn=61&lineEndColumn=76&lineStyle=plain&_a=contents

Reply:
Nont and I worked together to correct the Tenant ID issue, as well as update the go.mod file.